README.md

```diff
@@ -3,7 +3,7 @@ RehabJs
 
 [![Build Status](https://travis-ci.com/wix-incubator/rehabjs.svg?token=ECstxpHzEZeQwCfcirJ4&branch=master)](https://travis-ci.com/wix-incubator/rehabjs)
 [![NPM Version](https://img.shields.io/npm/v/rehabjs.svg?style=flat)](https://www.npmjs.com/package/rehabjs)
-
+[![Discord Chat](https://img.shields.io/discord/308323056592486420.svg)](https://discord.com/channels/735444524783894528)
```

Comment:
@Bunakof, cool
It's the right way.
However, please check the number code of the discord channel. It seems it should be `(https://img.shields.io/discord/735444524783894528?style=flat-square)` instead of `(https://img.shields.io/discord/308323056592486420.svg)`

In addition, I just enabled the badge option according to the video instruction from this tutorial https://shields.io/category/chat